### DISCUSSION TITLE
Cloud Debug (deprecated) vs ECS Exec Command

### DISCUSSION THREAD

**Discussion by choeflake** [aws/aws-toolkit-jetbrains#3136] [Q&A]
The experimental Cloud Debug feature is deprecated and will be removed within two months. The ECS Exec Command replaces this feature. But how does it replace the Cloud Debug? As far as I can see, with EXC Exec Command we can run commands on the ECS container and start interactive sessions. That's a nice feature!
But how do we update the application itself? And how to connect the Java debugger for example?

**Comment by n-block** [comment]
If you've found a solution to your question in the meantime, I would be very thankful if you'd share the solution here.

**Comment by joaosouz91** [comment]
Can someone involved please answer? It was a good feature and now it's removed. How can we debug the remote container through the ECS Exec Command?
